client/src/js/services/ScrollService.js

```diff
@@ -21,14 +21,16 @@ ScrollService.$inject = ['$location', '$anchorScroll', '$timeout'];
  */
 function ScrollService($location, $anchorScroll, $timeout) { 
   var scrollDelay = 0;
-
+  
   // Always scroll an additional 50 pixels to ensure element is visiable 
   $anchorScroll.yOffset = 50;
 
   /** 
+   * @deprecated
```

Comment:
I am pretty sure you are the only one who has really used this method.  Could you do a quick writeup sometime on how to properly use the `ScrollService`, potentially in a wiki?